src/kohlrahbi/collect_pruefis.py

```diff
@@ -40,11 +41,14 @@
             for item in get_all_paragraphs_and_tables(parent=doc):
                 if isinstance(item, Table) and does_the_table_contain_pruefidentifikatoren(table=item):
                     # check which pruefis
+                    if not item.row_cells(0)[-1].paragraphs[-1].text.startswith("Prüfidentifikator"):
+                        continue
                     seed = Seed.from_table(docx_table=item)
                     logger.info("Found a table with the following pruefis: %s", seed.pruefidentifikatoren)
-                    all_pruefis = all_pruefis + seed.pruefidentifikatoren
+                    for pruefi in seed.pruefidentifikatoren:
+                        all_pruefis.update({pruefi: ahb_file_path.name})
```

Comment:
```suggestion
                        all_pruefis.update({pruefi: ahb_file_path})
```
<-- nimm doch den path ist doch entspannter als mit strings zu arbeiten